ethcore/evm/src/interpreter/mod.rs

```diff
@@ -590,7 +596,7 @@ impl<Cost: CostType> Interpreter<Cost> {
 			},
 			instructions::EXTCODECOPY => {
 				let address = u256_to_address(&stack.pop_back());
-				let code = ext.extcode(&address)?;
+				let code = ext.extcode(&address)?.unwrap_or(Arc::new(Vec::new()));
```

Comment:
maybe `match` over here instead of `unwrap_or` ? I believe that we could avoid `Arc` and `Vec` allocations